core/src/test/scala/com/spotify/tfexample/derive/ExampleConverterTest.scala

```diff
@@ -46,7 +46,7 @@ class ExampleConverterTest extends FlatSpec with Matchers {
           .putFeature("string", stringFeat("world"))
           .build)
     featuresOf(actual) shouldEqual featuresOf(expected)
-    converter.fromExample(actual) shouldEqual record
+    converter.fromExample(actual).get shouldEqual record
```

Comment:
Would be great to see `Some(record)` on the right-hand side rather than these `.get`'s

Reply:
👍